modules/core/src/test/scala/dev/profunktor/redis4cats/codecs/DisciplineSuite.scala

```diff
@@ -21,16 +21,15 @@ import org.typelevel.discipline._
 
 trait DisciplineSuite extends ScalaCheckSuite {
 
-  def checkAll(name: String, ruleSet: Laws#RuleSet)(
-      implicit loc: Location
+  def checkAll(name: String, ruleSet: Laws#RuleSet)(implicit
+      loc: Location
```

Comment:
IMO, I think I prefer the old way where implicit is same level as variable.

Reply:
I agree , not a fan of splitting the implicit keyword from the rest